tests/functional_tests/ffront_tests/test_past_interface.py

```diff
@@ -0,0 +1,34 @@
+# GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2014-2021, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+from typing import Tuple
+
+from functional.common import Field
+from functional.ffront.decorator import field_operator
+from functional.ffront.fbuiltins import float64
+from functional.ffront.func_to_past import ProgramParser
+
+
+@field_operator
+def make_tuple_op(inp: Field[..., float64]) -> Tuple[Field[..., float64], Field[..., float64]]:
```

Comment:
While there are still places where `...` is used, we decided to avoid using it until we have parametric types. Just use a `Dimension`.